Shared/Samples/Edit geodatabase with transactions/EditGeodatabaseWithTransactionsView.swift

```diff
@@ -0,0 +1,266 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct EditGeodatabaseWithTransactionsView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The action currently being preformed.
+    @State private var selectedAction: Action? = .setUp
+    
+    /// The text describing the status of the sample.
+    @State private var statusText = ""
+    
+    /// The point on the map where the user tapped.
+    @State private var tapLocation: Point?
+    
+    /// A Boolean value indicating whether a transaction is active on the geodatabase.
+    @State private var isInTransaction = false
+    
+    /// A Boolean value indicating whether a transaction is required to add a feature.
+    @State private var transactionIsRequired = true
+    
+    /// A Boolean value indicating whether the select feature type popover is presented.
+    @State private var isSelectingFeatureType = false
+    
+    /// A Boolean value indicating whether the alert to end a transaction is presented.
+    @State private var endTransactionAlertIsPresented = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        MapView(map: model.map)
+            .onSingleTapGesture { _, mapPoint in
```

Comment:
When I tap on a blank space west to the Christmas Bay area, it shows in the error detail

> "Feature geometry is outside the generate geodatabase geometry. Encountered on feature with ID of -9223372036854775808."

While the error alert doesn't show much useful info.